README.md

```diff
@@ -7,155 +7,30 @@
 ```
 
 
-Use the same terminal for everything. The main reason for this plugin is to
-reuse the terminal easily. All commands open a terminal if one does not already
-exist. REPL commands open a terminal and the proper REPL if not already opened.
+Forked from:
+https://github.com/kassio/neoterm
 
-- Neovim/Vim terminal helper functions/commands.
-- Wraps REPLs to receive current file, line or selection.
-- Multiple terminal support:
-  - ![many-terms](https://cloud.githubusercontent.com/assets/120483/8921869/fe459572-34b1-11e5-93c9-c3b6f3b44719.gif)
 
-## Installation
+Branch demo:
+![1](https://raw.githubusercontent.com/incoggnito/neoterm/master/assets/sample.gif)
 
-### Vundle/Plug.vim/minpac/any other
-
-You can install this plugin using any vim plugin manager by using the path on
-GitHub for this repository:
 
+Some new settings:
 ```vim
-[Plugin|Plug|...] 'kassio/neoterm'
-```
-
-See the your plugin manager documentation for more information.
-
-### Manual
-
-For installation without a package manager, you can clone this Git repository
-into a bundle directory as with pathogen, and add the repository to your
-runtime path yourself. First clone the repository:
+let g:neoterm_default_mod = 'vertical'
 
-```console
-cd ~/.vim/bundle
-git clone https://github.com/kassio/neoterm.git
-```
+# use python in a virtual environment
+let g:neoterm_repl_python_venv = 'conda activate py3conda'
 
-Then, modify your `~/.vimrc` file to add this plugin to your runtime path:
+# set the repl
+let g:neoterm_repl_python = 'ipython'
 
-```vim
-set nocompatible
-filetype off
+# clear any startup screen like neofetchk
+let g:neoterm_clean_startup = 1
 
-let &runtimepath.=',~/.vim/bundle/neoterm'
+# use the paste magic command
+let g:neoterm_repl_ipython_magic = 1
 
-filetype plugin on
+# define the cell marker
+let g:neoterm_repl_cellmarker = '^# %%'
```

Comment:
where do these variables are being used?

Reply:
There are some changes missing. I try to fix it.

Reply:
The changes should be visible now. I will test again, when i've merged my branch with your actual version.